### ISSUE TITLE
Glenohumeral dysplasia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10008]
**Preferred term label:**
Glenohumeral dysplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Apert syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
this is rather a bundled term:
https://radiopaedia.org/cases/glenohumeral-dysplasia
Probably it is best not to make a single HPO term for this, but to consider "glenohumeral dysplasia" as a disease entity?

**Comment by MickeySegal** [comment]
We got it from an article on Apert syndrome:
https://www.ncbi.nlm.nih.gov/books/NBK541728/
"Restriction of movement involving the shoulder due to glenohumeral dysplasia can lead to functional impairment."

Although there is a disease entity with this name:
https://www.josefeichingermd.com/orthopaedic-surgeon-charleston-mount-pleasant-sc-patient-instruction-modules-glenoid-dysplasia.html
"While primary glenoid dysplasia is a rare developmental abnormality of the shoulder, the term glenoid dysplasia refers to malformations in the lower rim of the glenoid, the “socket” of the shoulder."

https://journals.lww.com/jaaos/abstract/2016/05000/glenoid_dysplasia.6.aspx
"Glenoid dysplasia is a developmental anomaly of the scapula that is characterized by a bony deficiency of the posteroinferior glenoid and the adjacent scapular neck."

**Comment by pnrobinson** [comment]
@MickeySegal makes sense

Adding new term: 
- ID: HP:6000934
- label: Glenoid dysplasia
- parents: [Abnormality of the glenoid fossa (HP:0011912)](https://hpo.jax.org/app/browse/term/HP:0011912)
- pmid: PMID:27055054
- def: Glenoid dysplasia is a developmental anomaly of the scapula that is characterized by a bony deficiency of the posteroinferior glenoid and the adjacent scapular neck.
- comment: Glenoid dysplasia may occur as a primary isolated condition or in association with various syndromes. It is thought to be related to defective ossification of the inferior glenoid precartilage. Radiographs typically demonstrate bilateral, symmetric dysplasia of the scapular neck with a range of associated bony changes. Glenoid dysplasia has been associated with instability and premature glenohumeral arthritis, although the clinical presentation is highly variable. Symptoms are delayed or mitigated as a result of compensatory glenoid labral hyperplasia.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10008
- synonyms: Glenohumeral dysplasia(NONE)
